docassemble/MarriageWithoutDelay/data/questions/marriage-without-delay.yml

```diff
@@ -111,27 +64,36 @@ code: |
   Marriage_without_Delay_preview_question
   signature_date
   basic_questions_signature_flow
-  users[2-1].signature
   Marriage_without_Delay = True
   download
   comments_to_clerk
 ---
-need: Marriage_without_Delay
+code: |
+  signature_fields = ["users[0].signature", "users[1].signature"]
```

Comment:
Signatures get asked for before actually showing the user the proper signature flow. I think it's because lines 102 and 103 should be in a `showif`.